terminus-ui/toggle/src/toggle.component.ts

```diff
@@ -104,7 +104,18 @@ export class TsToggleComponent extends TsReactiveFormBaseComponent {
    * Emit an event each time the toggle value changes
    */
   // TODO: Rename to avoid conflict with native events: https://github.com/GetTerminus/terminus-ui/issues/1472
```

Comment:
I think we can remove this. We have tickets to track it and it's deprecated in the codebase. 👍